src/worker/predictive-text-studio-worker-impl.ts

```diff
@@ -39,4 +50,69 @@ export class PredictiveTextStudioWorkerImpl
   updateBCP47Tag(bcp47Tag: string): Promise<void> {
     return this.storage.updateBCP47Tag(bcp47Tag);
   }
+
+  async generateKMPFromStorage(): Promise<Blob | void> {
```

Comment:
I like this method, but I think it should be `private` and **not** part of the App/Worker interface. 
```suggestion
  private async generateKMPFromStorage(): Promise<Blob | void> {
```